### PR TITLE
aks: add 'enable-ultra-ssd' parameter

### PR DESCRIPTION
**Pull request by andyzhangx**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
aks

File: src/aks-preview/azext_aks_preview/_params.py [hunk 1]
```diff
@@ -225,6 +226,7 @@ def load_arguments(self, _):
             c.argument('kubelet_config', type=str)
             c.argument('linux_os_config', type=str)
             c.argument('enable_encryption_at_host', options_list=['--enable-encryption-at-host'], action='store_true')
+            c.argument('enable_ultra_ssd', options_list=['--enable-ultra-ssd'], action='store_true')
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/_params.py:229]: The newly added parameter "enable_ultra_ssd" will be exposed as an option in hyphenated form (--enable-ultra-ssd) automatically, which is exactly the same as that specified in "options_list". Maybe you can omit this, or change the above  context ("aks create") to make them consistent .

Comment (andyzhangx) [reply src/aks-preview/azext_aks_preview/_params.py]: fixed.

**Review by FumingZhang — APPROVED** [review]

**Comment by andyzhangx** [comment]
@yonzhan could you help merge this PR? our AKS team has approved this PR.

**Comment by yonzhan** [comment]
@zhoxing-ms will help you.

File: src/aks-preview/azext_aks_preview/_params.py [hunk 2]
```diff
@@ -225,6 +226,7 @@ def load_arguments(self, _):
             c.argument('kubelet_config', type=str)
             c.argument('linux_os_config', type=str)
             c.argument('enable_encryption_at_host', options_list=['--enable-encryption-at-host'], action='store_true')
+            c.argument('enable_ultra_ssd', action='store_true')
```
Comment (zhoxing-ms) [inline src/aks-preview/azext_aks_preview/_params.py:229]: Do we need to consider adding `is_preview=True` for this parameter?

Comment (andyzhangx) [reply src/aks-preview/azext_aks_preview/_params.py:229]: no, we will GA this feature this month.
And actually cli-extension is preview command, so it does not matter.

File: src/aks-preview/HISTORY.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ Release History
 0.5.15
 +++++
 * Update to use 2021-05-01 api-version
+* Add argument '--enable-ultra-ssd'
```
Comment (zhoxing-ms) [inline src/aks-preview/HISTORY.md:9]: Please specify the feature of argument `--enable-ultra-ssd`, such as `Add argument '--enable-ultra-ssd' to enable UltraSSD on agent node pool`

**Review by zhoxing-ms — CHANGES_REQUESTED** [review]
Could you add some test cases for this new parameter?

File: src/aks-preview/azext_aks_preview/_params.py (hunk 2 above)
Comment (zhoxing-ms) [reply src/aks-preview/azext_aks_preview/_params.py:229]: OK, got it

File: src/aks-preview/HISTORY.md (hunk 3 above)
Comment (andyzhangx) [reply src/aks-preview/HISTORY.md]: fixed

**Comment by andyzhangx** [comment]
> Could you add some test cases for this new parameter?

@zhoxing-ms e2e test is added, could you approve? thanks.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**